HackingWithSwift/SwiftUI/Project2/Guess the Flag/Guess the Flag/ContentView.swift

```diff
@@ -6,6 +6,31 @@
 //
 
 import SwiftUI
+struct FlagImage: View {
+    var flagName: String
+    
+    var body: some View {
+        Image(flagName)
+            .clipShape(.capsule)
+            .shadow(radius: 10)
+    }
+}
+
+struct Title: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .font(.largeTitle)
+            .foregroundStyle(.yellow)
+            .bold()
+            .shadow(radius: 5)
+    }
+}
+
+extension View {
+    func titleStyle () -> some View {
+        modifier(Title())
+    }
+}
```

Comment:
It's preferably to extract that type of code (independent views, extensions and modifiers) into its own, separate files.